tests/unit/utils/resource-helper-test.js

```diff
@@ -0,0 +1,66 @@
+import Ember from 'ember';
+import sinonTest from 'ember-sinon-qunit/test-support/test';
+import { module } from 'qunit';
+
+import loadAssets from 'ember-cli-bundle-loader/utils/load-assets';
+import ResourceHelper from 'ember-cli-bundle-loader/utils/resource-helper';
+
+
+module('Unit | Utility | resource loader');
+
+sinonTest('loadJavascript adds the appropriate script tag to the DOM', function (assert) {
+
+  let testAsset = 'assets/load-assets-test2.js';
+
+  assert.equal(false, ResourceHelper.isJavascriptLoaded(testAsset));
+
+  return ResourceHelper.loadJavascript(testAsset).finally(function() {
```

Comment:
Is there a reason to use finally here instead of then? Since we're expecting it to be successful, checking then seems more appropriate, but I may be missing something